game/src/gui/foundation/key_event.h

```diff
@@ -11,6 +11,10 @@ class KeyEvent {
  public:
   enum class State : uint16_t { kIdle, kDown, kUp };
   enum class Mod : uint16_t { kNone = 0, kCtrl = 1, kShift = 2, kAlt = 4 };
+  enum {
+    ESC = 27,
```

Comment:
According to our enum rules, this should be `kEsc`.

Reply:
OK